src/crimson/osd/pg.h

```diff
@@ -557,6 +559,9 @@ class PG : public boost::intrusive_ref_counter<
 				       ObjectContextRef&& obc,
 				       ceph::os::Transaction&& txn,
 				       const osd_op_params_t& oop);
+  seastar::future<> rep_repair_primary_object(Ref<MOSDOp> m, 
```

Comment:
how shall i interpret this method name? how about "repair_object()"?

i understand it comes from `PrimaryLogPG::rep_repair_primary_object()`.  but the question still holds.

Reply:
If we want to rename, leaving a trace for grep with the original name (even a comment would do) seems desirable.

Reply:
'rep' - replicated (since we're doing replicated recovery)
'repair_primary_object' - We're recovering (repairing) the missing object of the primary pg

This is how I interpreted it, at least according to the implementation in ceph-osd. Either way, I don't mind changing it to 'repair_object()` and leave a comment if you want.

Reply:
i will merge this change as it is. will create another PR to do the rename. as i don't think we should expose the internals using the name of interface, and there is no need to differentiate `rep_repair_primary_object()` from another repair method.